addons/mrp/migrations/10.0.2.0/pre-migration.py

```diff
@@ -148,6 +148,8 @@ def rename_mrp_workorder(cr):
 
 
 def prepopulate_fields(cr):
+    if openupgrade.column_exists(cr, 'mrp_production', 'picking_type_id'):
+        return False
```

Comment:
why `return False` instead of `return` alone?

Reply:
Well, it's not very important as this is going to work the same returning `None` (default) than `False`, so I merge.